src/main/java/com/sobytylnik/hello/MyTest.java

```diff
@@ -0,0 +1,60 @@
+package com.sobytylnik.hello;
```

Comment:
According to Maven's project structure, tests should be located NOT in sources directory `src/main/java`, but in test directory `src/test/java`.